ansible/roles/aws/tasks/create.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Create new EC2 Instance
+  amazon.aws.ec2:
+    key_name: cdoan-jenkins-key-2
+    instance_type: m4.2xlarge
+    image: ami-066df92ac6f03efca # RHEL 8.2
+    wait: yes
+    group: launch-wizard-1
+    exact_count: 5
+    vpc_subnet_id: subnet-817895db
+    assign_public_ip: yes
+    region: us-west-1
+    volumes:
+      - device_name: /dev/sda1
+        volume_size: 100
+        volume_type: gp2
+        delete_on_termination: true
```

Comment:
Some of these could become vars even if we don't adjust all frequently, some who have to adjust with who is running the playbook. We could add them to maybe `vars/aws.yml` to "namespace" them from the other playbooks since it seems there will be a lot of them.

Reply:
Thats right. Let me expose them as variables now.